includes/Elements/Team_Member.php

```diff
@@ -913,12 +913,12 @@ protected function register_controls() {
 	protected function render( ) {
 
         $settings = $this->get_settings_for_display();
-		$team_member_image = $settings['eael_team_member_image'];
+		$team_member_image = $settings['eael_team_member_image'] ?? null;
```

Comment:
@himadree bhai let's use `''` instead of null (in both places)
We have 0 occurrences of null in php files. 
We are mostly using `''`